### PR TITLE
Improvements and tweaks for the rainbow laser

### PR DESCRIPTION
**Pull request by ultraplen**
I did a PR a few months back (#13320) that was "a little" half baked. This PR hopefully improves the rainbow laser from the why-did-i-push-this teir it is on now. 

 * The laser has been given a new sprite, with a color scheme based off the bike horn.

 * The laser has been given in-hand sprites

 * The flavour text has been tweaked

 * The braindamage firemode has been removed as it was neither useful or interesting.

New sprite:
dreamseeker_2017-07-30_19-50-42
(The orange light is a slightly more orange shade than in this screenshot) edit: the laser is laying right next to a grey line which makes the barrel look off. Better pic:
gimp-2 8_2017-07-30_20-21-57

Old sprite:
dreamseeker_2017-07-30_19-56-06

The laser is still unobtainable outside bus. 

:cl:

 * balance: Removed the braindamage firemode from the rainbow laser because it was dumb and turned frogs gay.

### CODE REVIEW

**Comment by jknpj** [comment]
Actually you can obtain it through xenoarch

**Comment by ultraplen** [comment]
Cool, I didn't know about that. Is it because of some xenoarch mechanic or did someone manually add it in to some loot table ?

**Comment by jknpj** [comment]
They were added to the random energy gun loot

**Comment by Riekland** [comment]
I much prefer the design of the old sprite
If you could keep the level of quality in the new one but put back the dorsal energy thing and make it change colour that would be lovely

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -376,29 +376,13 @@
 /obj/item/weapon/gun/energy/laser/rainbow
 
 	name = "rainbow laser"
-	desc = "A fearsome gun used by clown special forces. Its design is flawed however as clumsy users find it hard to operate."
+	desc = "The NanoTrasen iniative to develop a laser weapon for clowns was a failure as the intended users were to clumsy to operate them."
```
Comment (Intigracy) [inline code/modules/projectiles/guns/energy/laser.dm]: `too clumsy`

**Review by Intigracy — DISMISSED** [review]

**Comment by gbasood** [comment]
It's silly but could you add a changelog saying you removed the braindamage mode? Could even just be
"Removed braindamage from rainbow laser because it was useless and dumb and turned the frogs gay"

**Comment by ultraplen** [comment]
@Riekland 

I tried a bunch of different energy-meters and they all ended up looking p shitty. Here is a quick version with a dorsal energy-meter. The blue bar ruins the shape of the thing

gimp-2 8_2017-07-31_13-59-47

**Comment by Blithering** [comment]
Maybe moving the meter to the side of the gun, rather than the top would look better

**Comment by ultraplen** [comment]
I tried that aswell but it didnt come out good at all

**Pull request merged**